spec/defines/resource_server_spec.rb

```diff
@@ -125,6 +125,18 @@
               value: 45,
               match: %r{\s+listen\s+\[::\]:45 default ipv6only=on;}
             },
+            {
+              title: 'should set the IPv6 listen port',
```

Comment:
You might want to change the title to 'should set the IPv6 listen port range'.

Reply:
Updated. Thanks for pointing it out!